apps/network-alerts/js/notification/notification.js

```diff
@@ -18,12 +18,17 @@ function onNotification(message) {
     return;
   }
 
-  var title = Utils.parseParams(message.imageURL).titleID;
+  var title = null;
+  if (message.data) {
+    title = message.data.title;
+  }
+  else {
+    title = Utils.parseParams(message.imageURL).titleID;
```

Comment:
__question:__ I think we don't need this anymore, why do you think we still need it? If it's just for the backward compatibility - don't worry and just keep ```message.data.title`` :)